### PR TITLE
Feature/split button dropdown css style

### PR DESCRIPTION
**Pull request by captureteam**
I (trond) think this way of writing the css is more readable, but I guess it's a matter of taste.

Advantages:
space between >/+ and selector indicates that the operator is not something on the selector but a relationship towards the selector above.
explicitly using & in the child selector I think adds to the readability

### CODE REVIEW

**Comment by b2l** [comment]
I prefer this style too, it's easier to read for me.
LGTM!

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 1]
```diff
@@ -93,9 +93,9 @@ category: Components
 
 .btn-group {
 
-  >.btn {
+  & > .btn {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 2]
```diff
@@ -93,9 +93,9 @@ category: Components
...
 
-    +.dropdown-toggle {
+    & + .dropdown-toggle {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 3]
```diff
@@ -105,13 +105,13 @@ category: Components
     padding-right: $gutter-horizontal;
   }
 
-  >.btn-primary {
+  & > .btn-primary {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 4]
```diff
@@ -105,13 +105,13 @@ category: Components
...
 
     &.btn-split {
       border-right-color: $white;
     }
 
-    +.dropdown-toggle {
+    & + .dropdown-toggle {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 5]
```diff
@@ -123,13 +123,13 @@ category: Components
     box-shadow: none;
   }
 
-  >.btn-default.dropdown-toggle {
+  & > .btn-default.dropdown-toggle {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 6]
```diff
@@ -123,13 +123,13 @@ category: Components
...
     background: $btn-default-bg;
     border-color: $btn-default-border-active;
     color: $btn-default-color-active;
   }
 
-  >.btn-primary.dropdown-toggle {
+  & > .btn-primary.dropdown-toggle {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 7]
```diff
@@ -152,9 +152,9 @@ category: Components
   padding-top: 20px;
   right: -5px;
 
-  >li {
+  & > li {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

File: src/mnd-bootstrap/components/_button_dropdowns.scss [hunk 8]
```diff
@@ -152,9 +152,9 @@ category: Components
...
 
-    >a {
+    & > a {
```
Comment (mnd-hound) [inline src/mnd-bootstrap/components/_button_dropdowns.scss]: Unnecessary parent selector (&)

**Pull request merged**